### ISSUE TITLE
I'm pretty sure this should be in the translations file as opposed to being hardcoded...

### ISSUE THREAD

**Issue by shadow93** [50DKP/FF2-Official#258]
One wonders why the goomba stomp texts are hardcoded and not in the translations file...

note the hardcoded goomba stomp/stomped text?

``` sourcemod
public Action:OnStomp(attacker, victim, &Float:damageMultiplier, &Float:damageBonus, &Float:JumpPower)
{
    if(!Enabled || !IsValidClient(attacker) || !IsValidClient(victim) || attacker==victim)
    {
        return Plugin_Continue;
    }

    if(IsBoss(attacker))
    {
        if(shield[victim])
        {
            new Float:position[3];
            GetEntPropVector(attacker, Prop_Send, "m_vecOrigin", position);

            TF2_RemoveWearable(victim, shield[victim]);
            EmitSoundToClient(victim, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
            EmitSoundToClient(victim, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
            EmitSoundToClient(attacker, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
            EmitSoundToClient(attacker, "player/spy_shield_break.wav", _, _, _, _, 0.7, _, _, position, _, false);
            TF2_AddCondition(victim, TFCond_Bonked, 0.1);
            shield[victim]=0;
            return Plugin_Handled;
        }
        damageMultiplier=900.0;
        JumpPower=0.0;
        PrintHintText(victim, "Ouch!  Watch your head!");
        PrintHintText(attacker, "You just goomba stomped somebody!");
        return Plugin_Changed;
    }
    else if(IsBoss(victim))
    {
        damageMultiplier=GoombaDamage;
        JumpPower=reboundPower;
        PrintHintText(victim, "You were just goomba stomped!");
        PrintHintText(attacker, "You just goomba stomped the boss!");
        UpdateHealthBar();
        return Plugin_Changed;
    }
    return Plugin_Continue;
}
```

**Comment by WildCard65** [comment]
I believe when I made the basic support I had no clue how translations work.

**Comment by naydef** [comment]
I think this can be closed, not to forget to merge with FF2 2.0
